tests/fixtures/typescript-imports/.eslintrc.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   plugins: ['self'],
-  extends: ['plugin:self/typescript', 'plugin:self/node'],
+  extends: ['plugin:self/node', 'plugin:self/typescript'],
```

Comment:
This doesn't seem right - typescript is a core config, and node is an augment so the ordering should be extending fro, typescript then node

Reply:
oof, good catch!